File: TF10. METHODS. DEB AND TROUBLESHOOTING CODE - EX (Java)/src/P11_GeometryCalculator.java
import java.util.Scanner;

public class P11_GeometryCalculator {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        String figureType = in.nextLine();
        double result;

        switch (figureType) {
            case "triangle":

                double a = Double.parseDouble(in.nextLine());
                double b = Double.parseDouble(in.nextLine());

                result = GetTriangleArea(a, b);
                System.out.printf("%.2f", result);
                break;

            case "square":

                double c = Double.parseDouble(in.nextLine());

                result = GetSquareArea(c);
                System.out.printf("%.2f", result);
                break;

            case "rectangle":

                double d = Double.parseDouble(in.nextLine());
                double e = Double.parseDouble(in.nextLine());

                result = GetRectangleArea(d, e);
                System.out.printf("%.2f", result);
                break;

            case "circle":

                double r = Double.parseDouble(in.nextLine());

                result = GetCircleArea(r);
                System.out.printf("%.2f", result);
                break;
        }

    }

    private static double GetCircleArea(double r) {
        return Math.PI * Math.pow(r, 2);
    }

    private static double GetRectangleArea(double a, double b) {
        return a * b;
    }

    private static double GetSquareArea(double a) {
        return Math.pow(a, 2);
    }

    private static double GetTriangleArea(double a, double b) {
        return (a * b) / 2;
    }
}

File: TF10. METHODS. DEB AND TROUBLESHOOTING CODE - EX/P11.GeometryCalculator.cs
using System;

namespace P11.Geometry_Calculator
{
    class Program
    {
        static void Main(string[] args)
        {
            string figureType = Console.ReadLine();
            double result;

            switch (figureType)
            {
                case "triangle":

                    double a = double.Parse(Console.ReadLine());
                    double b = double.Parse(Console.ReadLine());

                    result = GetTriangleArea(a, b);
                    Console.WriteLine($"{result:f2}");
                    break;

                case "square":

                    double c = double.Parse(Console.ReadLine());

                    result = GetSquareArea(c);
                    Console.WriteLine($"{result:f2}");
                    break;

                case "rectangle":

                    double d = double.Parse(Console.ReadLine());
                    double e = double.Parse(Console.ReadLine());

                    result = GetRectangleArea(d, e);
                    Console.WriteLine($"{result:f2}");
                    break;

                case "circle":

                    double r = double.Parse(Console.ReadLine());

                    result = GetCircleArea(r);
                    Console.WriteLine($"{result:f2}");
                    break;

                default:

                    throw new ArgumentException();
            }

        }

        private static double GetCircleArea(double r)
        {
            double area = Math.PI * Math.Pow(r, 2);
            return area;
        }

        private static double GetRectangleArea(double a, double b)
        {
            double area = a * b;
            return area;
        }

        private static double GetSquareArea(double a)
        {
            double area = Math.Pow(a, 2);
            return area;
        }

        private static double GetTriangleArea(double a, double b)
        {
            double area = (a * b) / 2;
            return area;
        }
    }
}
